src/components/FeelPopupButton.js

```diff
@@ -0,0 +1,32 @@
+import { PopupIcon } from '../components/icons';
+import classNames from 'classnames';
+
+/**
+ * A reusable button component for opening popups.
+ *
+ * @param {Object} props
+ * @param {Function} props.onClick - Callback to trigger when the button is clicked.
+ * @param {string} [props.title] - Tooltip text for the button.
+ * @param {boolean} [props.disabled] - Whether the button is disabled.
+ * @param {string} [props.className] - Additional class names for the button.
+ */
+export function FeelPopupButton({
```

Comment:
:spiral_notepad: I moved the popup button into its own component, minor thing for consistency.